controllers/model/grafana_resources.go

```diff
@@ -11,6 +11,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	v2 "sigs.k8s.io/gateway-api/apis/v1"
```

Comment:
Please, use consistent naming for imports (e.g. `gwapiv1` like in another file).